app/tests/routes/wallet/dids/test_create_did.py

```diff
@@ -1,21 +1,33 @@
 from unittest.mock import AsyncMock, patch
 
 import pytest
-from aries_cloudcontroller import DIDCreate
+from aries_cloudcontroller import DIDCreate as AcapyDIDCreate
 from aries_cloudcontroller.exceptions import (
     ApiException,
     BadRequestException,
     NotFoundException,
 )
 
+from app.models.wallet import DIDCreate
 from app.routes.wallet.dids import create_did
 
 
 @pytest.mark.anyio
 @pytest.mark.parametrize(
-    "request_body", [None, DIDCreate(method="key"), DIDCreate(method="sov")]
+    "request_body, create_body",
+    [
+        (None, None),
+        (
+            DIDCreate(method="key"),
+            AcapyDIDCreate(method="key", options={"key_type": "ed25519"}),
+        ),
+        (
+            DIDCreate(method="sov"),
+            AcapyDIDCreate(method="sov", options={"key_type": "ed25519"}),
+        ),
+    ],
```

Comment:
The unit tests are cheap so it's good to cover almost every scenario. May as well branch this out to cover more method/key_type combinations